### ISSUE TITLE
Do you plan to support Counter Strike 2 (CS:GO on Source 2) on SM/MM?

### ISSUE THREAD

**Issue by IL0co** [alliedmodders/sourcemod#1958]
# Help us help you
  - [+] I have checked that my issue [doesn't exist yet](https://github.com/alliedmodders/sourcemod/issues).

I think this is one of the main questions today. Will SourceMod support Counter Strike 2 (CS:GO on Source 2)?

This version was recently announced for summer 2023.
https://counter-strike.net/cs2

**Comment by IL0co** [comment]
> https://wiki.alliedmods.net/Introduction_to_SourceMod_Plugins#Will_SourceMod_support_Source_2.3F_Will_plugins_for_existing_games_continue_to_work_if_they_are_ported.3F

Yes, I saw this, but perhaps something has already changed during this time (the post was supposedly in 2017).

**Comment by zer0k-z** [comment]
It was changed a few days ago.

**Comment by rumblefrog** [comment]
Nothing has changed, the last revision of that text was on 13 March 2023‎, less than 10 days ago.

Things like these are discussions for the forum or Discord, not on the issue tracker.
